scene/3d/soft_body_3d.cpp

```diff
@@ -361,6 +361,12 @@ void SoftBody3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_drag_coefficient", "drag_coefficient"), &SoftBody3D::set_drag_coefficient);
 	ClassDB::bind_method(D_METHOD("get_drag_coefficient"), &SoftBody3D::get_drag_coefficient);
 
+	ClassDB::bind_method(D_METHOD("get_point_transform", "point_index"), &SoftBody3D::get_point_transform);
+
+	ClassDB::bind_method(D_METHOD("pin_point_toggle", "point_index"), &SoftBody3D::pin_point_toggle);
```

Comment:
`pin_point_toggle` makes sense for the editor gizmo, but it seems redundant for scripts since there's already a function to set it to true or false, so I would rather remove this one.

Reply:
I don't see any downside to having this exposed. If someone just wants to toggle some pins, theres no need to rewrite logic thats been in the engine all this time.

Reply:
Yeah, you don't need to remove the method entirely from the engine, it's just not necessary to expose it to scripts. It can be confusing to have redundant functions, and especially this one is not that useful in game script context.

Reply:
Perhaps not so much for games, _at this time_,  but surely for addons. Godot's non-script tools for editing pinned points are quite limited. The pin click can be expanded to a sphere volume brush or selection box/lasso.

I don't understand how its confusing or redundant either. Without a toggle method, the logic to check, change, and set the pin flag would have to be made to recreate a toggle. And if reused multiple times, that logic would be turned into a method. That seems redundant to me.

In any case, if you still don't agree, I won't press the matter further.

Reply:
Given that the same thing can be done with `set_point_pinned(point_index, !is_point_pinned(point_index))` I can't see a good reason to bloat the script API with an extra function.

The only cases where a helper function would be justified is if it's not trivial, like some math that users would have to figure out otherwise, or something that requires many lines of script.